src/util/colorScale.js

```diff
@@ -218,63 +221,31 @@ function createOrdinalScale(colorBy, t1nodes, t2nodes) {
   return {continuous, colorScale, legendValues, legendBounds};
 }
 
-function createContinuousScale(colorBy, providedScale, t1nodes, t2nodes, isTemporal) {
-  /* Note that a temporal scale is treated very similar to a continuous one... for the time being.
-     In the future it'd be nice to allow YYYY-MM-DD values, but that's for another PR (and comes
-     with its own complexities - what about -XX dates?)                     james june 2022    */
-  // console.log("making a continuous color scale for ", colorBy);
-  if (colorBy==="num_date") {
-    /* before numeric scales were a definable type, num_date was specified as continuous */
-    isTemporal = true;
-  }
+function createContinuousScale(colorBy, providedScale, t1nodes, t2nodes) {
+
   let minMax;
-  if (isTemporal) {
-    // empty - minMax not needed
-  } else if (colorBy==="lbi") {
+  if (colorBy==="lbi") {
     minMax = [0, 0.7]; /* TODO: this is for historical reasons, and we should switch to a provided scale */
   } else {
     minMax = getMinMaxFromTree(t1nodes, t2nodes, colorBy);
   }
 
   /* user-defined anchor points across the scale */
-  const anchorPoints = _validateContinuousAnchorPoints(providedScale);
+  const anchorPoints = _validateAnchorPoints(providedScale, (val) => typeof val==="number");
 
   /* make the continuous scale */
   let domain, range;
   if (anchorPoints) {
     domain = anchorPoints.map((pt) => pt[0]);
     range = anchorPoints.map((pt) => pt[1]);
-  } else if (isTemporal) {
-    /* we want the colorScale to "focus" on the tip dates, and be spaced according to sampling */
-    let rootDate = getTraitFromNode(t1nodes[0], colorBy);
-    let vals = t1nodes.filter((n) => !n.hasChildren)
-      .map((n) => getTraitFromNode(n, colorBy));
-    if (t2nodes) {
-      const treeTooRootDate = getTraitFromNode(t2nodes[0], colorBy);
-      if (treeTooRootDate < rootDate) rootDate = treeTooRootDate;
-      vals.concat(
-        t2nodes.filter((n) => !n.hasChildren)
-          .map((n) => getTraitFromNode(n, colorBy))
-      );
-    }
-    vals = vals.sort();
-    domain = [rootDate];
-    const n = 10;
-    const spaceBetween = parseInt(vals.length / (n - 1), 10);
-    for (let i = 0; i < (n-1); i++) domain.push(vals[spaceBetween*i]);
-    domain.push(vals[vals.length-1]);
-    domain = [...new Set(domain)]; /* filter to unique values only */
-    range = colors[domain.length]; /* use the right number of colours */
   } else {
     range = colors[9];
     domain = genericDomain.map((d) => minMax[0] + d * (minMax[1] - minMax[0]));
   }
   const scale = scaleLinear().domain(domain).range(range);
 
   let legendValues;
-  if (isTemporal) {
-    legendValues = domain.slice(1);
-  } else if (colorBy==="lbi") {
+  if (colorBy==="lbi") {
     /* TODO: this is for historical reasons, and we should switch to a provided scale */
     legendValues = [0, 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7];
```

Comment:
Not related at all to this PR, but whoa...I had no idea we hardcoded the LBI scale in Auspice. Can we (I?) remove this in a separate PR?

Reply:
> I had no idea ...

Once upon a time 😉 https://github.com/nextstrain/auspice/issues/805

Reply:
Oh no...

Reply:
@jameshadfield https://github.com/nextstrain/auspice/issues/541

I should stop pulling on this thread :D

Reply:
ha! too good